docs/cosmos-hub/delegator-guide-cli.md

```diff
@@ -358,6 +358,24 @@ The transaction `fees` are the product of `gas` and `gasPrice`. As a user, you h
 For mainnet, the recommended `gas-prices` is `0.025uatom`. 
 ::: 
 
+### Sending Tokens
+
+::: tip
+**Before you can bond atoms and withdraw rewards, you need to [set up `gaiacli`](#setting-up-gaiacli) and [create an account](#creating-an-account)**
+:::
+
+::: warning
+**Note: These commands need to be run on an online computer. It is more secure to perform them commands using a ledger device. For the offline procedure, click [here](#signing-transactions-from-an-offline-computer).**
+::: 
+
+```bash
```

Comment:
Maybe we should just point to the `gaiacli.md` docs? The `develop` branch has changed this to now:

```
$ gaiacli tx send [from_key_or_address] [to_address] [amount] [flags]
```

Reply:
I think we should keep it as what it is in mainnet, BUT I will update it when doing the cherry pick to develop (setting a reminder rn)

Reply:
Did we kill the `--from` flag?

Reply:
Yeah but not on v0.34.x yet, which is what the master docs reflect.